pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -62,6 +62,7 @@ type TransportServer struct {
 // TransportServerSpec is the spec of the TransportServer resource.
 type TransportServerSpec struct {
 	Listener           TransportServerListener `json:"listener"`
+	Snippets           string                  `json:"snippets"`
```

Comment:
Could we name it `serverSnippets`?  (both go field and YAML field) because such snippets apply to the `server` context. This would be consistent with the snippets for HTTP resources - http, server and location snippets.